netbox/dcim/tables/devices.py

```diff
@@ -557,6 +557,12 @@ class BaseInterfaceTable(NetBoxTable):
         orderable=False,
         verbose_name=_('IP Addresses')
     )
+    virtual_ips = tables.TemplateColumn(
```

Comment:
I suggest renaming this to `fhrp_group_ips` or similar, as "virtual" IPs may not be quite clear to the user.